launch_testing_ros/README.md

```diff
@@ -0,0 +1,28 @@
+# launch\_testing\_ros
+
+## Examples
+
+### `talker_listener_launch_test.py`
+
+Usage:
+
+```sh
+launch_test test/examples/talker_listener_launch_test.py
+```
+
+This test launches the talker and listener example nodes from demo\_nodes\_py and interacts
+with them via their ROS interfaces.  Remapping rules are used so that one of the tests can sit in
```

Comment:
New sentence should start on new line.

Same below.

Reply:
Oops. It predates the PR but you're right.